src/Altinn.App.Api/Controllers/ActionsController.cs

```diff
@@ -156,7 +156,7 @@ public class ActionsController : ControllerBase
     private async Task SaveChangedModels(Instance instance, Dictionary<string, object?> resultUpdatedDataModels)
     {
         var instanceIdentifier = new InstanceIdentifier(instance);
-        foreach (var (dataType, newModel) in resultUpdatedDataModels)
+        foreach (var (elementId, newModel) in resultUpdatedDataModels)
```

Comment:
The comment about what the system should return states 

https://github.com/Altinn/app-lib-dotnet/blob/3f2dbd03700122cf9f59d3350686010f73513900/src/Altinn.App.Core/Models/UserAction/UserActionResult.cs#L17-L20

But I'm really happy if frontend used the more sensible elementId from the start. It removes one paragraph from https://github.com/Altinn/app-lib-dotnet/issues/446

Reply:
Huh, I didn't know - that comment must be wrong. The key is, and must be, the elementId. I changed this here because existing user actions broke. :thinking:

Reply:
If it broke existing actions that is really great 🎉🎉🎉🎉

Much easier to fix the comment than to make kompatibility with both options.

Reply:
Yeah, sorry my bad! Comment and code is really misleading here, the key is elementId and always was so this is just a renaming luckily!